lib/rubocop/cop/style/bitwise_predicate.rb

```diff
@@ -71,6 +71,7 @@ class BitwisePredicate < Base
         PATTERN
 
         def on_send(node)
+          return unless node.receiver
           return unless node.receiver.begin_type?
```

Comment:
These two lines can be combined: 

```suggestion
          return unless node.receiver&.begin_type?
```

Reply:
Sure, was not sure how big you all were on the safe navigation operator :)